### DISCUSSION TITLE
Problem in local setup Lern Module

### DISCUSSION THREAD

**Discussion by devaj04** [Sunbird-Lern/Community#77] [Issues] [answered]
URL is not working - https://github.com/project-sunbird/sunbird-lms-mw/blob/master/actors/src/main/resources/cassandra.cql
Unable to find -  /tmp/cql/pageMgmt.csv and  /tmp/cql/pageSection.csv
Unable to run file - cqlsh -f /sunbird-lms-service/service/src/main/resources/cassandra.cql

**Comment by BharathwajShankar** [comment, marked as answer]
@devaj04 please kindly follow the below link for the local setup of lern module,
(https://lern.sunbird.org/use/developer-installation/user-and-org-service/installation-guide)
Kindly check whether you have set up everything properly with environment variables and also check that the cql file has proper executing permission in your system.
